content/sensu-go/5.15/release-notes.md

```diff
@@ -41,6 +44,81 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.15.0 release notes
+
+**November 18, 2019** &mdash; The latest release of Sensu Go, version 5.15.0, is now available for download. This is a significant release for a number of reasons. The first being the changes to licensing, making 100% of Sensu Go's commercial features available for free to all users, up to your first 100 entities! This release also includes the long awaited Cluster Federation features, supporting multi-cluster authentication, RBAC policy replication, and a single pane of glass for your Sensu monitoring data! We added support for API keys, making it easy to integrate with the Sensu API (you no longer need to manage JWTs!). In addition, the 5.15.0 release includes support for sensu-backend environment variables and bug fixes that improve error logging for mutator execution and flap detection weighting for checks.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.15.0.
+
+**IMPORTANT:**
+Sensu's free entity limit is now 100 entities. All [commercial features][79] are available for free in the packaged Sensu Go distribution up to an entity limit of 100. You will see a warning when you approach the 100-entity limit (at 75%).
+
+If your Sensu instance includes more than 100 entities, [contact us][90] to learn how to upgrade your installation and increase your limit. See [Discourse][91] for more information about our usage policy.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][79]) Added support for the [federation cluster registration API][85].
```

Comment:
There are a few other federation components that ship in 5.15.0 (see https://github.com/sensu/sensu-enterprise-go/blob/master/CHANGELOG.md#unreleased). The cluster registration API is just one of those things. We could either list all components, or roll them all up into a single federation feature.